### PR TITLE
Added React Native to Readme

### PR DESCRIPTION
**Pull request by Fritsch-Tech**
#### 🔧 Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Codebase improvement
- [ ] Other (if none of the other choices apply)

#### 🚨 Checklist

- [x] I've read the [guidelines for contributing](https://github.com/gremo/react-directus/blob/main/CONTRIBUTING.md)
- [x] I've added necessary documentation (if appropriate)
- [x] I've ensured that my code additions do not fail linting or unit tests (if applicable)

#### Description

Added information for the use of React Native in the Readme.

Resolving #141

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,42 @@ export const TodoList = () => {
 };
 ```
 
+## 📱 React Native
```
Comment (gremo) [inline README.md]: Please move this section after components

File: README.md [hunk 2]
```diff
@@ -40,14 +40,14 @@ import React from 'react';
...
-  <DirectusProvider apiUrl="https://api.example.com">
+  <DirectusProvider apiUrl="https://api.example.com" options={{}}>
     <App />
   </DirectusProvider>,
   document.getElementById('root')
 );
 ```
 
-You can optionally pass an `apiOptions` object to the provider, it will be passed to the client as the [`init`](https://docs.directus.io/reference/sdk/#reference) parameter.
+You can optionally pass an `options` object to the provider, it will be passed to the directus instance as the [`init`](https://docs.directus.io/reference/sdk/#reference) parameter.
```
Comment (gremo) [inline README.md]: I think this is not part of this PR right?

File: README.md [hunk 3]
```diff
@@ -40,14 +40,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 ReactDOM.render(
-  <DirectusProvider apiUrl="https://api.example.com">
+  <DirectusProvider apiUrl="https://api.example.com" options={{}}>
```
Comment (gremo) [inline README.md]: I think this is not part of this PR right?

File: README.md (hunk 2 above)
Comment (Fritsch-Tech) [reply README.md]: I saw it while adding the React-Native section and decided to just update it in this PR. But i can revert the changes and open a new one for them if you like.

**Comment by Fritsch-Tech** [comment]
I moved the "React Native" section, moved the changes to the "Quick start" to a seperate pullrequest and sqashed the commits.

File: README.md (hunk 2 above)
Comment (gremo) [reply README.md]: Yes!
